apps/cloudfront-functions/vite.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from 'vitest/config';
+
+export default defineConfig({
+  test: {
+    watch: false,
+    coverage: {
+      provider: 'v8',
+      reporter: ['text', 'json', 'html'],
+      thresholds: {
+        branches: 70,
+        lines: 70,
+      },
```

Comment:
In the IDE, I see an error, the 'thresholds' property doesn't exist. 
![Screenshot 2024-01-11 at 16 17 25](https://github.com/pagopa/b2b-portals/assets/4775679/db78db49-c497-48c7-acf3-3a583d8bfe5d)
![Screenshot 2024-01-11 at 16 11 12](https://github.com/pagopa/b2b-portals/assets/4775679/c44c41ba-398e-47b5-a1a2-380d5f655730)

Looking at the types, it should work with the following structure:

```suggestion
      branches: 70,
      lines: 70,
```

Reply:
Actually it's fine, I hadn't updated the vitest version correctly 😅🙏